test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -60,6 +60,36 @@ def create_mom_resources(self, resc_name,
                                         validate=True)
         self.scheduler.add_resource(resc_name)
 
+        return fp
+
+    def create_mult_mom_resources(self, resc_name,
+                                  resc_type, resc_flag,
+                                  script_body):
+        """
+        helper function for create multiple mom_resources.
+        """
+        fp_list = []
+
+        resc_name_mom = ",".join(resc_name)
+        resc_name_mom = '"' + resc_name_mom + '"'
+        resc_name_svr = ",".join(resc_name)
+        self.scheduler.set_sched_config({'mom_resources': resc_name_mom},
+                                        validate=True)
+        self.scheduler.add_resource(resc_name_svr)
+
+        for i in range(2):
```

Comment:
can you explain why you are looping 2 times

Reply:
number of elements in the list are 2 so i am iterating 2 times. now i am first caluculate the length of list and then iterate that much time.